### ISSUE TITLE
Improve importing of plain CSS

### ISSUE THREAD

**Issue by AlecRust** [TrySound/postcss-easy-import#10]
I'm looking to import plain CSS files (like `normalize.css`) in amongst other [SugarSS](https://github.com/postcss/sugarss) imports. Is this something that can be improved in this module?

Something like:

``` css
@import "normalize.css"
@import "base/base.sss"
@import "components/button.sss"
etc.
```

Currently my output is:

![screenshot](https://i.imgur.com/DXGteCP.png)

Please see https://github.com/postcss/sugarss/issues/39, https://github.com/postcss/sugarss/issues/33

**Comment by simonsmith** [comment]
Fixed in `9.0.0` of `postcss-import` - https://github.com/postcss/postcss-import/pull/245

Will be making a release here with v9 shortly

**Comment by AlecRust** [comment]
Sweet!
